### PR TITLE
Add/home

### PR DESCRIPTION
**Pull request by saviomuniz**
Primeira versão da home finalizada em desktop, tablet e phone

### CODE REVIEW

**Comment by arturpimentel** [comment]
I got this in [https://savio--extensions.myvtex.com/](https://savio--extensions.myvtex.com/):
image
Is that correct? 😬

**Comment by afonso-praca** [comment]
I got this in the mobile for iphone 7

image
I've linked the app in this branch -> https://afonso--extensions.myvtex.com/

**Comment by afonso-praca** [comment]
In the mobile version, the testimonials does not looked to be respecting the space between each one. On the design file it should be 64px. Also the name of the person looks too far from the comment itself.

image

**Comment by afonso-praca** [comment]
In the mobile version I also perceived too much space between the last testimonial and the following section.

image

**Comment by afonso-praca** [comment]
Ii the ipad version, the top menu is bleeding horizontally, so the page shows an horizontal scroll. Other components in the page are also bleeding and with their sizes calculated based on the width larger than the device viewport.

In the testimonials line should we really show the arrows? Is it possible to just show half of the second card, as proposed in the layout?

image

**Comment by afonso-praca** [comment]
In what regards the typograph, we are facining some issue to use the proposed type?

File: store/blocks/header.jsonc [hunk 1]
```diff
@@ -7,30 +7,74 @@
...
     ]
   },
+  "flex-layout.col#spacer": { 
+    "props": { 
+      "width": "grow"
+    }
+  },
+  "login": {
+    "props": {
+      "emailAndPasswordTitle": "LOGIN",
```
Comment (afonso-praca) [inline store/blocks/header.jsonc:47]: i did not saw this reflecting in the layout. may i am missing something.

File: store/blocks/header.jsonc [hunk 2]
```diff
@@ -7,30 +7,74 @@
...
     "children": [
-      "flex-layout.row#transparent"
+      "flex-layout.row#header"
     ]
   },
   "header-layout.mobile": { 
     "children": [
-      "flex-layout.row#transparent"
+      "responsive-layout.tablet#header-tablet",
+      "responsive-layout.phone#header-tablet"
```
[the remark this answers is not in the document]
Comment (saviomuniz) [reply store/blocks/header.jsonc:16]: yup, thanks

**Comment by saviomuniz** [comment]
> In what regards the typograph, are we facing some issue to use the proposed type?

I mentioned this in our meetings that I would leave this to the end, because it's something that can only be done on the Site Editor and I'll lose those changes once I go to another workspace

**Comment by saviomuniz** [comment]
> I got this in the mobile for iphone 7
> 
> image
> I've linked the app in this branch -> https://afonso--extensions.myvtex.com/
> It's the same reported by @arturpimentel. An extra margin on left and right.
> Should the `sign in` text be `login`?

Yup, we'll need to override the message, we'll do this once we go to a production workspace, otherwise will be lost. Same thing as the font. 

About the margin, I commented that on above 👆

**Comment by saviomuniz** [comment]
> Ii the ipad version, the top menu is bleeding horizontally, so the page shows an horizontal scroll. Other components in the page are also bleeding and with their sizes calculated based on the width larger than the device viewport.
> 
> In the testimonials line should we really show the arrows? Is it possible to just show half of the second card, as proposed in the layout?
> 
> image

About the bleeding: have you reloaded the page once gone to the tablet version? You need to do that, otherwise you won't see all the responsive tablet content. In my browser that behavior is not happening. 

About the testimonies: there's no way to do that currently, already talked with the developer that made the slider layout. I discussed that with Pedro and he told me to implement that way, not sure about displaying the arrows though.

**Comment by saviomuniz** [comment]
> Are we missing a layout for the drawer menu content?
> 
> image

Drawer is not implemented yet

**Comment by saviomuniz** [comment]
> In the mobile version, the testimonials does not looked to be respecting the space between each one. On the design file it should be 64px. Also the name of the person looks too far from the comment itself.
> 
> image

Nice catch

**Comment by saviomuniz** [comment]
> In the mobile version I also perceived too much space between the last testimonial and the following section.
> 
> image

Nice catch 👍

**Pull request merged**